### PR TITLE
feat: added a readme to profile

### PR DESCRIPTION
**Pull request by Mino03947**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,2 +1,118 @@
...
+
+## Requirements
+
+**Platforms**: iOS 17.0.3+ (Apple devices) and Android 12+ (Samsung devices).<br>
+**Location Data**: Limited to users in the Netherlands.
+
+## Project Timeline
+
+**Start Date**: 16 August 2023<br>
+**End Date**: December 2023
```
Comment (JaronZ) [inline profile/README.md:35]: This date is not up-to-date and has already passed.

File: profile/README.md [hunk 2]
```diff
@@ -1,2 +1,118 @@
...
+
+## License
+
+This project is licensed under the MIT License - see the [LICENSE.md](LICENSE.md) file for details.
+
+## Contributors
+
+- [Mino03947](Mino03947)
+- [Jaronline](Jaronline)
+- [Altergith](Altergith)
```
Comment (JaronZ) [inline profile/README.md:113]: When I click on these links I see a 404 page.
Should this link to our github profiles?

File: profile/README.md [hunk 3]
```diff
@@ -1,2 +1,118 @@
...
+
+## Getting Started
+
+To run the project locally, clone the repository:
+
+```bash
+git clone https://github.com/compairifuel/compairifuel.git
+```
+
+For detailed setup and installation instructions, refer to the [INSTALL.md](INSTALL.md) file.
```
Comment (JaronZ) [inline profile/README.md:103]: I see a 404 page when clicking on this link

File: profile/README.md [hunk 4]
```diff
@@ -1,2 +1,118 @@
...
+
+## License
+
+This project is licensed under the MIT License - see the [LICENSE.md](LICENSE.md) file for details.
```
Comment (JaronZ) [inline profile/README.md:107]: I see a 404 page when clicking on this link

**Review by altergith — APPROVED** [review]
Purpose and goals of the project are clear, nicely done.

**Review by JaronZ — CHANGES_REQUESTED** [review]

**Review by altergith — APPROVED** [review]
HUZZAH

**Pull request merged**